back/src/domain/convention/useCases/broadcast/BroadcastToPoleEmploiOnConventionUpdates.unit.test.ts

```diff
@@ -21,7 +22,9 @@ const prepareUseCase = async ({
   const poleEmploiGateWay = new InMemoryPoleEmploiGateway();
   const uow = createInMemoryUow();
   uow.featureFlagRepository = new InMemoryFeatureFlagRepository({
-    enablePeConventionBroadcast,
+    enablePeConventionBroadcast: makeBooleanFeatureFlag(
+      enablePeConventionBroadcast,
```

Comment:
peut-être revoir ce nommage :
```suggestion
      enablePeConventionBroadcastIsActive,
```

Reply:
enablePeConventionBroadcast contient toutes les données de feature flag, dont le `isActive` (mais aussi le `kind`) dans ce cas là